### DISCUSSION TITLE
MonitorControl using a slinger slider for each function

### DISCUSSION THREAD

**Discussion by J-u-n-o** [emoacht/Monitorian#159] [General]
Hello,

Created a similar tool myself, though with a different approach:
Two slider for contrast and brightness to control all monitors. And also synchronize the changes made on the external monitor using the monitor keys, and laptop keyboard for the internal monitor.
It is using a calibration/mapping table to compensate for different monitor characteristics. These can be changed using the GUI and a graph.
All nicely written in python and QT.
Known issue: system resume requires restarting of the tool, and the setup should use tabpages instead of a 'scrolling' widget (information; brightness; contrast)
Curious about your opinion.

https://github.com/J-u-n-o/MonitorControl

**Comment by emoacht** [comment]
Thank you for the information.
It is quite interesting and I am happy to explore your repository when I have time.

**Comment by emoacht** [comment]
I moved this topic to Discussions.
